index.html

```diff
@@ -56,8 +58,16 @@
         /></a>
       </nav>
     </header>
+    <section class="mobile-menu">
+      <a href="#" class="close-btn" id="close-menu-btn">X</a>
+      <ul class="a-color">
+         <li><h3>Portfolio</h3><a class="listBtns" href="#profile">></a></li>
+         <li><h3>About</h3><a class="listBtns"  href="#about">></a></li>
+         <li><h3>Contact</h3><a class="listBtns"  href="#contact">></a></li>
```

Comment:
- [ ] Please  remove those `>` 
![Screenshot from 2021-10-05 00-40-38](https://user-images.githubusercontent.com/62705947/135934202-2706eb35-def5-48bb-8ae7-5f9747caba45.png)
![Screenshot from 2021-10-05 00-40-38](https://user-images.githubusercontent.com/62705947/135934431-22233628-98e4-4142-a60c-fb83fb3c94ee.png)

e

Reply:
The requirement has been fulfilled 👏🏼